project/eScavenge/CLI.py

```diff
@@ -106,32 +106,24 @@ def remove_landmark(self, args):
 
 @need_admin
 def start(self, _):
-    try:
-        self.game.start()
-    except IndexError:
-        return invalid_param
+    self.game.start()
     if self.game.started:
         return game_started
     return "Failed to start Game"
 
 
 @need_admin
 def end(self, _):
-    try:
-        self.game.end()
-    except IndexError:
-        return invalid_param
+    self.game.end()
     if self.game.ended:
         return game_ended
     return "Failed to end game"
 
 
 @need_admin
 def create(self, _):
-    try:
+    if self.game is None or not self.game.started or self.game.ended:
```

Comment:
Minor comment, instead of is None you could use not self.game
They are equivalent, so no real need to change it, I think the python “preferred” way would be the latter since it doesn’t rely on a hard coded value

Reply:
It ends up failing 87 out of 92 tests if I do it that way.  I tried a variety of ways and this seemed to be what the compiler liked.